portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -108,4 +113,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Redirect the user back to the Comments page, which shows all the added comments
     response.sendRedirect("/#comments");
   }
+
+  /**
+   * Returns a value between -1 and 1, representing how negative or positive text is
+   */
+  private static float getSentimentScore(String text) throws IOException {
+    // Create a new Document that contains the parameter - text - as content
+    Document doc =
+        Document.newBuilder().setContent(text).setType(Document.Type.PLAIN_TEXT).build();
+
+    LanguageServiceClient languageService = LanguageServiceClient.create();
```

Comment:
It's better to call it "client" instead of "service" :)

Reply:
Done, thanks!